corehq/toggles/__init__.py

```diff
@@ -2995,13 +2995,13 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     namespaces=[NAMESPACE_DOMAIN],
 )
 
-INCREASE_DEVICE_LIMIT_PER_USER = StaticToggle(
```

Comment:
Nobody has this turned on, right?

Reply:
Correct. And if they do, they shouldn't 😅